static/src/javascripts/projects/commercial/modules/article-body-adverts.js

```diff
@@ -96,10 +99,11 @@ const addDesktopInlineAds = (isInline1: boolean): Promise<number> => {
         filter: filterNearbyCandidates(adSizes.mpu.height),
     };
 
+    // For any other inline
     const relaxedRules = {
         bodySelector: '.js-article__body',
         slotSelector: ' > p',
-        minAbove: 1000,
+        minAbove: 1000 + (isPaidContent ? 600 : 0),
```

Comment:
minor / bikesheddding: slight preference for following the pattern on line 82:

`minAbove: isImmersive ? 700 : 300,`

that would make this:

`minAbove: isPaidContent ? 1600 : 1000,`